gammapy/utils/time.py

```diff
@@ -0,0 +1,30 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Time related utility functions."""
```

Comment:
Please add the dummy `from __future__ import ...` line, even if it doesn't matter here at the moment.